.env.test

```diff
@@ -79,4 +79,8 @@ DOI_PREFIX=https://doi.org/10.62891
 # Cross ref api 
 CROSSREF_API=https://api.crossref.org
 CROSSREF_API_KEY=
-CROSSREF_EMAIL=
\ No newline at end of file
+CROSSREF_EMAIL=
+
+# Automated metadata
+AUTOMATED_METADATA_API=
+AUTOMATED_METADATA_API_KEY=
```

Comment:
what does this api key refer to?

Reply:
Secret key used to authenticate api request between desci-server and automated metadata server